src/components/CandidateForm.jsx

```diff
@@ -42,21 +60,29 @@ const CandidateForm = ({ fields, onSubmit }) => {
   return (
     <>
       <form onSubmit={formHandler} className={styles.form}>
-        {values.map((field) => (
-          // 4. Check if field is not technologies
-          <div key={field.name} className={styles.field}>
-            <input
-              required
-              onChange={onChange}
-              value={field.value}
-              placeholder={field.placeholder || ""}
-              name={field.name}
-            ></input>
-          </div>
+        {values.map(
+          (field) =>
+            // 4. Check if field is not technologies
+            field.name !== "technologies" ? (
```

Comment:
Instead of the ternary, you could do `field.name !== "technologies" && <div>...</div>`